src/configs/types.ts

```diff
@@ -0,0 +1,37 @@
+import { z } from 'zod';
+
+export const zConfigSchema = z.object({
+  id: z.string(),
+  version: z.string(),
+  value: z.record(z.string(), z.unknown()),
+});
+
+export type Config = z.infer<typeof zConfigSchema>;
+
+export enum ConfigSpecialVersion {
```

Comment:
can you use the one already in `src/util.ts` (and rename it to `SpecialVersion` or something)

https://github.com/autoblocksai/javascript-sdk/blob/060e0a0feaf7b03deebd44653ff8930a55f6ae7e/src/util.ts#L28-L31